sklearn/linear_model/_coordinate_descent.py

```diff
@@ -979,7 +979,6 @@ def fit(self, X, y, sample_weight=None, check_input=True):
                 accept_sparse="csc",
                 order="F",
                 dtype=[np.float64, np.float32],
-                force_writeable=True,
```

Comment:
I think the removal of this line and the other `force_writeable=True` lines below were not intentional (maybe when resolving conflicts with `main`)?

I think this is what causes the test failures on the CI.